core/conversion/var/Var.cpp

```diff
@@ -85,6 +85,27 @@ std::string Var::type_name() const {
   }
 }
 
+nvinfer1::ITensor* Var::ITensorOrFreeze(ConversionCtx* ctx, const torch::jit::Node* n) {
```

Comment:
Why does this take a node as input?

Reply:
Used for assigning the constant layer's name

Reply:
Id rather leave the node out honestly even if it makes the error slightly less helpful (hopefully context can cover for us)